public/app/screens/main/sections/settings/user-avatar/style.scss

```diff
@@ -0,0 +1,28 @@
+@import "theme";
+
+.cc-user-avatar{
+  @include grid-row(collapse);
+
+  position: relative;
+  margin: convert-to-rem(8px);
+
+  .cc-user-avatar__image-content {
+    @include grid-column(10);
+    .cc-user-avatar__image{
+      width: convert-to-rem(85px);
```

Comment:
font sizes definitely should be in `rem`s
in general, it depends on whether you wish to scale/resize things independently of others (fixed-size)